### DISCUSSION TITLE
Qiita CLIで記事を執筆後、git pushしたらworkflowでエラーが発生する

### DISCUSSION THREAD

**Discussion by tadashi1105** [increments/qiita-discussions#914] [ヘルプ] [answered]
初めまして、@tadashi1105と申します。

workflowで以下のエラーが発生しました。解決方法をご存知でしたら教えていただきたいです。

```
error: pathspec 'by' did not match any file(s) known to git
error: pathspec 'qiita-cli' did not match any file(s) known to git
Error: Process completed with exit code 1.
```

<img width="1419" alt="スクリーンショット 2024-10-09 18 57 56" src="https://github.com/user-attachments/assets/04b98c53-1f3b-4c0d-b1d6-6e26f4591657">

添付画像の`git commit -m Updated by qiita-cli`の箇所の"Updated by qiita-cli"がクオートで囲まれていないのが気になる点です。

コメント先が不適切でしたら教えていただけると幸いです。

**Comment by ohakutsu** [comment, marked as answer]
@tadashi1105
ご報告ありがとうございます。

調査したところ、Qiita CLI v1.6.0で発生する問題でした :bow:
ご不便をおかけして申し訳ございません。

こちらの問題を修正したv1.6.1をリリースしましたので、ご報告いただいた問題は解消されると思われます。
https://github.com/increments/qiita-cli/releases/tag/v1.6.1

もし、変わらずエラーが発生しているようでしたら、ご報告いただけると幸いです :pray:

**Reply by tadashi1105** [reply]
@ohakutsu 
ご対応いただきありがとうございます。

解決し、workflowが問題なく完了しました。
